### PR TITLE
Add env verifier params to applicable file urls

### PR DESCRIPTION
**Pull request by olegrise**
## Description
Add 3 env verifier parameters to the file request URL

## Motivation and Context
Env verifier requirements - analyze data usage based on env verifier parameters.

## How Has This Been Tested?
- Added unit tests
- Used Charles proxy to run it locally, then confirmed in Chrome inspector that env verifier parameters were added to the file URL.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/rise-image.js [hunk 1]
```diff
@@ -430,7 +430,21 @@ class RiseImage extends WatchFilesMixin( ValidFilesMixin( RiseElement )) {
   }
 
   _getFileUrl( file ) {
-    return RiseImage.STORAGE_PREFIX + this._encodePath( file ) + "?_=" + this._timeCreatedFor( file );
+    let url = RiseImage.STORAGE_PREFIX + this._encodePath( file ) + "?_=" + this._timeCreatedFor( file );
+
+    if (RisePlayerConfiguration.Helpers.getViewerType && RisePlayerConfiguration.Helpers.getViewerType()) {
```
Comment (stulees) [inline src/rise-image.js:435]: @olegrise In my PR https://github.com/Rise-Vision/web-component-rise-storage/pull/104 I'm naming the query params `viewerType`, `viewerEnv`, and `viewerId`. I think we should be consistent. Thoughts?

Comment (olegrise) [reply src/rise-image.js:435]: @stulees good point. I'll fix. I was actually looking at your source code to make sure they are the same :).

Comment (olegrise) [reply src/rise-image.js:435]: @stulees fixed. Please review.

**Comment by olegrise** [comment]
@stulees I can now see my file requests in BQ and env verifier parameters are there. All good.

**Review by stulees — APPROVED** [review]
@olegrise Looks good!

**Pull request merged**